packages/git/src/browser/diff/git-diff-widget.tsx

```diff
@@ -413,11 +428,20 @@ export class GitDiffListContainer extends React.Component<GitDiffListContainer.P
 
     componentDidMount(): void {
         this.props.addDiffListKeyListeners(this.props.id);
+        this.props.onListContainerReady(this.listContainer);
```

Comment:
WDYT of renaming `onListContainerReady` –> `setListContainer`? `onSomething` sounds like an event handler.

Reply:
Sounds good.

Reply:
Although, it actually is kind of an event handler.